pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "BootstrapReport"
-version = "1.0.0" 
+version = "1.0.1" 
```

Comment:
Are the changes here backwards-compatible? If not then per [symver](https://semver.org/) we may need to make a major version change. In that case we might want to leave the version number as is for now (for development purposes) and then increment when we are ready for the next release (maybe when shiny is live.)

Reply:
They are not backwards-compatible. Changed back to `1.0.0` in 60b7fea!